actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -26,6 +27,8 @@ def options
           headless_chrome_browser_options
         when :headless_firefox
           headless_firefox_browser_options
+        else
+          @option
```

Comment:
```suggestion
          capabilities
```

Reply:
that way you always get an „empty“ options object initialized by that method. i opted to return nil in case the user does not supply a block.
do we want useless object creation?